unittest/src/unittest_bucket_array.cpp

```diff
@@ -216,7 +216,7 @@ TEST_CASE("bucket_array test", "[bucket_array]")
 		while (count--)
 		{
 			CHECK(*it == i++);
-			it = next(it);
+			it = cpprelude::next(it);
```

Comment:
why?!

Reply:
without specifying the namespace, it had some compilation errors at resolving which namespace to use.